ambry-api/src/main/java/com/github/ambry/account/ContainerBuilder.java

```diff
@@ -29,6 +29,7 @@
   private short id;
   private String name;
   private ContainerStatus status;
+  private long lastUpdateTime;
```

Comment:
we can give lastUpdateTime a default value
```
private long lastUpdateTime = System. currentTimeMillis();
```
This way we don't have to change the constructor's signature.

Reply:
Updated.